about.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About
+    </title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" />
+    <script defer src="https://cdn.tailwindcss.com"></script>
+    <script defer src="tailwind.js"></script>
+
+    <script defer src="script.js"></script>
+</head>
+
+<body>
+    <!--header section-->
+    <header class="fixed z-50 w-screen">
+        <ul class="hidden md:flex md:px-52 h-14 bg-zinc-900 items-center justify-end text-white gap-8">
+            <li class="text-[1.2rem]">
+                <a href="#">
+                    <i class="fa-brands fa-facebook-f"></i>
+                </a>
+            </li>
+            <li class="text-[1.2rem]">
+                <a href="#">
+                    <i class="fa-brands fa-twitter"></i>
+                </a>
+            </li>
+            <li class="text-[1.2rem]">
+                <a href="#">English</a>
+            </li>
+            <li class="text-[1.2rem]">
+                <a href="#">MyPage</a>
+            </li>
+            <li class="text-[1.2rem]">
+                <a href="#">Logout</a>
+            </li>
+        </ul>
+        <nav class="h-20 p-5 md:px-50 bg-white/50 backdrop-blur-xl text-black flex items-center drop-shadow">
+            <div class="hidden lg:flex lg:px-48 w-full items-center justify-between">
+                <div class="w-14 h-auto ml-[10rem]">
+                    <a href="./index.html">
+                        <img src="./assets/img/heart.black.png" alt="website logo"
+                            class="object-contain w-full h-full hover:scale-125 transition duration-500">
+                    </a>
+                </div>
+                <div class="flex gap-20 items-center">
+                    <ul class="flex items-center gap-9">
+                        <li class="text-[1.2rem] text-gray-600 hover:text-orange-600 transition duration-200">
+                            <a href="./about.html">About</a>
+                        </li>
```

Comment:
Kindly update this navigation link on the about page so that it points to the home page, since right now I have no way to go back to the home page 🤗